offchain-modules/sdk/src/cell_collector.rs

```diff
@@ -106,6 +106,18 @@ pub fn get_capacity_cells_for_mint(
     get_live_cells(indexer_client, search_key, terminator)
 }
 
+pub fn get_all_live_cells_by_lockscript(
+    indexer_client: &mut IndexerRpcClient,
+    lockscript: Script,
+) -> Result<Vec<Cell>, String> {
+    let search_key = SearchKey {
+        script: lockscript.into(),
+        script_type: ScriptType::Lock,
+        args_len: None,
+    };
+    get_live_cells(indexer_client, search_key, |_, _| (true, true))
```

Comment:
This can only get the first live cell of lockscript. We should pass a `terminator` to ` fn get_live_cells` rather than `(trure, true)`